pyaerocom/data/resources.py

```diff
@@ -9,7 +9,8 @@
 if sys.version_info >= (3, 11):
     from importlib import resources
 else:
-    import importlib_resources as resources
+    # import importlib_resources as resources
+    from importlib import resources
```

Comment:
Possibly breaking CI?

Reply:
`importlib.resources` changed on Python 3.11 and deprecated most of the methods.
On Python 3.8 to 3.10 you need `importlib_resources>=5.10`

https://github.com/metno/pyaerocom/blob/09f244374971dd9a1721157e1b8eec92ea7ae11d/pyproject.toml#L25

Reply:
I did this so it would run in 3.10 at all. I could not run the reader with the original code

Reply:
you need to merge `dev-main` into your branch and recreate your connda environment, or install `importlib_resources` into it